package.json

```diff
@@ -22,20 +22,21 @@
     "url": "https://www.ferreiro.me/"
   },
   "scripts": {
-    "check": "npm-check",
-    "lint": "standard",
     "build:bower": "./node_modules/bower/bin/bower install",
     "build:gulp": "gulp",
     "build": "npm run build:gulp && npm run build:bower",
-    "watch": "gulp watch",
+    "check": "npm-check",
+    "lint": "",
+    "client": "cd web-app && PORT=5000 yarn start",
+    "server": "DEBUG=ferreiro:* nodemon --exec babel-node ./bin/www",
+    "dev": "concurrently --kill-others-on-fail \"yarn server\"  \"yarn build\"  \"yarn client\" \"yarn watch\"",
```

Comment:
do you need to `gulp build` & `gulp watch` in the same command. My first though would be that `watch` is running `build` continuously.